### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User did not agree to telemetry"

### DISCUSSION THREAD

**Discussion by lee1008** [community/community#13937] [Copilot Conversations]
GitHub Copilot could not connect to server. Extension activation failed: "User did not agree to telemetry"

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by tgbhy** [reply]
Don't work for me

**Reply by WISH12345** [reply]
didnt work for me too

**Reply by Peatechen** [reply]
vs code re-install copilot extension solved my problem

**Comment by amit-artlist** [comment]
Disable the addon -> reload VSCode -> popup will show up -> agree to the telemetry

**Comment by 201flaviosilva** [comment]
GitHub Copilot is now paid 🤑 : https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/
